script/deb-post-inst.template

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+command -v deb-systemd-invoke > /dev/null
+has_systemd=$?
+
+if [ $has_systemd -ne 0 ]; then
+	deb-systemd-invoke --daemon-reload
+fi
+
+if [ $has_systemd -ne 0 ] && [ -n "$(systemctl list-units --plain --no-legend {{.Service}})" ]; then
```

Comment:
```suggestion
if [ $has_systemd -eq 0 ] && deb-systemd-helper was-enabled {{.Service}} ; then
```

or something like that.

Reply:
`{{Service}}` can containg globbing, e.g. `syncthing@*.service`. Thus `is-enabled` doesn't work. Of course I totally agree that it is ugly, so I am very open to better ideas, just couldn't come up with them myself.